Google.Api.Gax/UnparsedResourceName.cs

```diff
@@ -12,30 +12,30 @@ namespace Google.Api.Gax
     /// <summary>
     /// A resource name in which nothing is known about the name structure.
     /// </summary>
-    public sealed class UnknownResourceName : IResourceName, IEquatable<UnknownResourceName>
+    public sealed class UnparsedResourceName : IResourceName, IEquatable<UnparsedResourceName>
     {
         /// <summary>
-        /// Parse a resource name into an <see cref="UnknownResourceName"/>.
+        /// Parse a resource name into an <see cref="UnparsedResourceName"/>.
```

Comment:
This sentence is kinda funky - "We're parsing it but we're not really." I think it's okay though.

Reply:
ack